### PR TITLE
3505 electronic delivery

### LINKED ISSUE #3505: Users cannot request electronic delivery for ReCAP Locations
### Expected behavior
When a user visits the request form for an item that offers both physical and electronic delivery from ReCAP they should be able to request other options. 

### Actual behavior
visit https://catalog.princeton.edu/requests/99115215833506421?aeon=false&mfhd=22703340470006421

### Steps to replicate
Visit any request form for a recap item. Try and select an item for electronic delivery, fill in the metadata fields required, the Request button at the button page never activates, it does activate when you select physical delivery. 

A mult-item example: https://catalog.princeton.edu/requests/994246483506421?aeon=false&mfhd=22618136940006421

### Impact of this bug
No work around. 

### Implementation notes
Happens on both single and multi-item holdings. Appears to be only ReCAP locations, for example this Firestone item is fine: https://catalog.princeton.edu/requests/99118269913506421?aeon=false&mfhd=22722909750006421, multi-item holding is also at https://catalog.princeton.edu/requests/99126836834306421?aeon=false&mfhd=22961622130006421.

### PR DESCRIPTION
**Pull request by christinach**
closes #3505

### CODE REVIEW

**Comment by christinach** [comment]
examples I tested:
https://catalog-qa.princeton.edu//requests/99115215833506421?aeon=false&mfhd=22703340470006421

https://catalog-qa.princeton.edu/requests/99110528583506421?aeon=false&mfhd=22732237860006421

https://catalog-qa.princeton.edu/requests/99115215823506421?aeon=false&mfhd=22703374900006421

https://catalog-qa.princeton.edu/requests/99122304923506421?aeon=false&mfhd=22511126440006421

https://catalog-qa.princeton.edu/requests/99126743805306421?aeon=false&mfhd=22954825980006421

https://catalog-qa.princeton.edu/requests/9928652563506421?aeon=false&mfhd=22709024380006421

 https://catalog-qa.princeton.edu/requests/998574693506421?aeon=false&mfhd=22579850750006421

https://catalog-qa.princeton.edu/requests/99126743805306421?aeon=false&mfhd=22954825980006421

https://catalog-qa.princeton.edu/requests/99118269913506421?aeon=false&mfhd=22722909750006421

File: spec/features/requests/request_spec.rb [hunk 1]
```diff
@@ -1271,17 +1271,18 @@
             check('requestable_selected_23641620980006421')
             expect(page).to have_button('Request Selected Items', disabled: false)
           end
-          it 'enables and disables for a record with user supplied volume data' do
+          it 'button state does not change when user supplies volume data or not' do
```
Comment (christinach) [inline spec/features/requests/request_spec.rb:1274]: Discussed with @kevinreiss to remove the user supply requirement.

**Comment by christinach** [comment]
please squash when merging

**Review by kevinreiss — APPROVED** [review]
I've tested our major scenarios for requests with this PR.

**Pull request merged**